### PR TITLE
Update README + Fix examples formatting

### PR DESCRIPTION
**Pull request by NadieFiind**

### CODE REVIEW

**Comment by NadieFiind** [comment]
Doesn't really change anything in the code. Just fixes some formatting and adds typing for more readablity.

File: examples/counter-app/src/index.py [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
 
 		return Element("main", lambda: [
 			Button(
 				onclick=decrement,
-				children=lambda: [ Text("-") ]
+				children=lambda: [Text("-")]
 			),
+			Text(" "),
 			Text(self.count),
+			Text(" "),
```
Comment (jabezborja) [inline examples/counter-app/src/index.py:25]: This is weird, you can just add styles to the `Text(self.count)` element.

**Review by jabezborja — CHANGES_REQUESTED** [review]

File: examples/counter-app/src/index.py (hunk 1 above)
Comment (NadieFiind) [reply examples/counter-app/src/index.py:25]: How do you do that? Can you give me an example?

Comment (jabezborja) [reply examples/counter-app/src/index.py:25]: I think you can do it with this:

```py
s = Style(padding_left="5px", padding_right="5px")

Text(self.count, styles=[s])
```

Comment (NadieFiind) [reply examples/counter-app/src/index.py:25]: `Text` is not an `Element` so we can't do that.`Text` is just a literal text in the HTML document without a tag. But we can still wrap it in an element to give it styling.

```py
s = Style(padding_left="5px", padding_right="5px")
Element("span", lambda: Text(self.count), styles=[s])
```

But it's a lot more code so I am not gonna do that.

Comment (jabezborja) [reply examples/counter-app/src/index.py:25]: LGTM. We can leave it for now, I'll just change it later.

**Pull request merged**